src/libraries/pkg/test/packageTest.targets

```diff
@@ -8,14 +8,7 @@
 
   <PropertyGroup>
     <ToolsDir>$(MSBuildThisFileDirectory)tools\</ToolsDir>
-
-    <RestoreSources>
-      https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json;
-      https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
-      https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json;
-      https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6-transport/nuget/v3/index.json;
-      $(LocalPackagesPath)
-    </RestoreSources>
+    <RestoreAdditionalProjectSources>$(LocalPackagesPath)</RestoreAdditionalProjectSources>
```

Comment:
nit: why not replace the LocalPackagesPath property with RestoreAdditionalProjectSources?

Reply:
That property isn't supposed to be passed in by the user. It's already passed in and by default points to the artifacts packages dir: https://github.com/dotnet/runtime/blob/6058cb7f2045a03dd4ba864a4963eeda12566260/src/libraries/pkg/test/testPackages.proj#L169.

Reply:
Good point. For some reason I thought it was for local testing, I forgot we generated the test restore command.

Reply:
Specifying known properties as global properties can make them effectively readonly in the evaluation, which could break other props/targets which were trying to append to them.  This was the case with RestoreSources previously.  I'm inclined to keep these seperate since it's not the intent of this to specify all additional restore sources, only one that we want to add.